test/flux_source_test.go

```diff
@@ -99,7 +100,8 @@ var _ = Describe("Flux source integration", func() {
 		// BeforeSuite so it's near where it's used.
 		createCRD.Do(func() {
 			// tell the client about CRDs
-			apiextensionsv1.AddToScheme(k8sClient.Scheme())
+			err := apiextensionsv1.AddToScheme(k8sClient.Scheme())
```

Comment:
`Expect(<reasonably short invocation>).To(Succeed())` is kind of the "house style", though admittedly not used everywhere.